content/posts/writing-posts/rich-content.md

```diff
@@ -32,12 +32,12 @@ Hugo ships with several [Built-in Shortcodes](https://gohugo.io/content-manageme
 
 ## Twitter Simple Shortcode
 
-{{< twitter_simple 1085870671291310081 >}}
+{{< tweet 1085870671291310081 >}}
```

Comment:
these i just updated to what the docs had while I was in here, let me know if there is a reason they need to be the `_simple` variant

Reply:
Just copied from the example site. So, no reason. :P